### DISCUSSION TITLE
Difference between max/min_element for simd and normal?

### DISCUSSION THREAD

**Discussion by ixru** [bitshifter/glam-rs#372] [Q&A] [answered]
```rust
assert_eq!(glam::Vec3::new(0.32, f32::NAN, f32::NAN).max_element(), 0.32);
assert_eq!(glam::Vec3A::new(0.32, f32::NAN, f32::NAN).max_element(), 0.32);
```
Top one is correct, bottom one is wrong. Is this intended? Wrong with Vec4 too, and correct with vec2, so is consistent. Can't find in documentation.

**Comment by bitshifter** [comment, marked as answer]
I'm not sure if this should be considered a bug or more just don't have any expectations when dealing with NaNs. The difference will be due to differences in implementation.

**Reply by ixru** [reply]
If you are asking, I had expectations created by seeing consistent behaviour, and found nothing telling me it was a bad assumption. Thank you for answer.
